index.js

```diff
@@ -48,96 +48,127 @@ function fromFile(file, callback) {
     });
 }
 
+
+function returnOutput(output) {
+    //check for tm2z
+    if (output.toString('ascii', 257, 262) === 'ustar') return 'tm2z';
+
+    //check for serial tiles
+    const head = output.slice(0, 50);
+    if (head.toString().indexOf('JSONBREAKFASTTIME') === 0) return 'serialtiles';
+
+    //check for tif+gz
+    if (
+        (output.slice(0, 2).toString() === 'II' || output.slice(0, 2).toString() === 'MM')
+        && ((output[2] === 42) || output[3] === 42 || output[2] === 43)) {
+        return 'tif+gz';
+    }
+
+    //default to unknown
+    throw invalid('Unknown filetype');
+}
+
+/**
+ * promisify zlib.gunzip.
+ *
+ * @param {buffer} buffer -
+ * @returns {promise}
+ */
+
+function gunzipPromise (buffer) {
+
+    if (buffer.Body) buffer = buffer.Body;
```

Comment:
@vincentsarago Curious what this case is for.

Reply:
yeah I copied that from another scripts, we can remove it sorry.